src/vehicle_use.cpp

```diff
@@ -1717,6 +1718,9 @@ void vehicle::interact_with( const tripoint &pos, int interact_part )
     if( turret.can_reload() ) {
         selectmenu.addentry( RELOAD_TURRET, true, 'r', _( "Reload %s" ), turret.name() );
     }
+    if( has_curtain ) {
+        selectmenu.addentry( PEEK_CURTAIN, true, 'p', _( "Peek through the closed curtains" ) );
```

Comment:
There is no check for the curtains being closed, so this will probably allow to peek through open curtains as well. That's not an error, but the entry may be confusing ("What *closed* curtain is the game talking about?").